.github/workflows/electron-win.yml

```diff
@@ -0,0 +1,54 @@
+name: Electron Rebuild CI - Windows (x32 & x64)
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        electron:
+          - 5
+          - 6
+          - 7
+          - 8
+        include:
+          # includes a new variable for each electron (essentially a switch statement) 
+          - electron: 5
+            nodeVersion: 70
+          - electron: 6
+            nodeVersion: 73
+          - electron: 7
+            nodeVersion: 75
+          - electron: 8
+            nodeVersion: 76
+        arch:
+          - ia32
+          - x64
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        os: ${{ matrix.os }}
+    - name: Install packages
+      run: npm install --unsafe-perm
+    - name: Install electron-rebuild
+      run: npm install --save-dev electron-rebuild@latest
+    - name: Run on ${{ matrix.os }} - Electron v${{ matrix.electron }} (Node ${{ matrix.nodeVersion }})
+      run: ./node_modules/.bin/electron-rebuild -v=${{ matrix.electron }} -w node-sass
+    - uses: actions/upload-artifact@v1
+      with:
+        name: win32_${{ matrix.arch }}_${{ matrix.nodeVersion }}_binding.node
+        path: build/release/binding.node
+    - uses: actions/upload-artifact@v1
+      with:
+        name: win32_${{ matrix.arch }}_${{ matrix.nodeVersion }}_binding.pdb
+        path: build/release/binding.pdb
+    # npm test failing on pre-existing master issues, commenting out
```

Comment:
If we figure out the test issues, the test step should probably be before the upload. If we end up just skipping it, then we can just delete this rather than comment it out